### ISSUE TITLE
CI failing with `buf` because "Failure: Remote generation alpha features have been sunset."

### ISSUE THREAD

**Issue by westonpace** [substrait-io/substrait#529]
Here is an example failing job: https://github.com/substrait-io/substrait/actions/runs/5527212791/job/14991956011?pr=518

The failure message links to a site with explanation:

```
Run buf generate
Failure: Remote generation alpha features have been sunset.

For more information, see https://buf.build/docs/migration-guides/migrate-remote-generation-alpha/
```

I'm assuming this is just a matter of tweaking our config.

**Comment by mbrobbel** [comment]
https://github.com/substrait-io/substrait/pull/499

**Comment by westonpace** [comment]
Ah, thanks for the pointer!

**Comment by mbrobbel** [comment]
Closed via #499.
